sql/core/src/main/scala/org/apache/spark/sql/UDFRegistration.scala

```diff
@@ -61,7 +61,7 @@ class UDFRegistration private[sql] (functionRegistry: FunctionRegistry) extends
         | dataType: ${udf.dataType}
       """.stripMargin)
 
-    functionRegistry.registerFunction(name, udf.builder)
+    functionRegistry.createOrReplaceTempFunction(name, udf.builder)
```

Comment:
Our current register function APIs in UDFRegistration does not allow users to specify the database names as part of `name`, since the registered functions are temporary.